### PR TITLE
COMMERCE-4789 making code consistent to calculate prices correctly in all content contributors

### PR DESCRIPTION
**Pull request by igorbeslic**
tnx

### CODE REVIEW

**Comment by igorbeslic** [comment]
ci:test:sf

**Comment by igorbeslic** [comment]
ci:test:relevant

**Comment by igorbeslic** [comment]
ci:stop

**Comment by igorbeslic** [comment]
ci:test:sf

**Comment by igorbeslic** [comment]
ci:test:relevant

**Comment by igorbeslic** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/price/test/CommerceProductPriceCalculationWithTaxV2Test.java [hunk 1]
```diff
@@ -185,14 +186,34 @@ public void testGetGrossProductPriceFromGrossPriceList() throws Exception {
 
 	@Test
 	public void testGetGrossProductPriceFromNetPriceList() throws Exception {
+		CommerceChannel commerceTaxIncludedChannel =
```
Comment (dsitu) [inline modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/price/test/CommerceProductPriceCalculationWithTaxV2Test.java:189]: @igorbeslic name should be like this.
```suggestion
		CommerceChannel taxIncludedCommerceChannel =
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**